v03_pipeline/lib/tasks/delete_family_tables.py

```diff
@@ -18,12 +19,14 @@ def complete(self) -> bool:
         )
 
     def run(self):
-        for family_guid in self.family_guids:
-            self.dynamic_delete_family_table_tasks.add(
-                DeleteFamilyTableTask(
-                    reference_genome=self.reference_genome,
-                    dataset_type=self.dataset_type,
-                    family_guid=family_guid,
-                ),
-            )
+        for sample_type in SampleType:
```

Comment:
potentially out of scope here, but I think we would actually want to be explicit about the sample type (or multiple sample types) we are deleting, so potentially we could delete one but not the other

Reply:
talked yesterday, this was intentional so that we won't have to remember to run the dag twice for both sample types